### DISCUSSION TITLE
Welcome to py3gpp Discussions!

### DISCUSSION THREAD

**Discussion by catkira** [catkira/py3gpp#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by catkira** [comment]
@vborchsh Do you think it makes sense to merge all PRs into a dev branch and only merge the dev branch into master when a new PyPi release is created? That could be a way to avoid a broken master state.

**Reply by vborchsh** [reply]
Sounds reasonable. Plus we can align CI to publish to pypi and gitpages only for master pushes.

**Reply by catkira** [reply]
can you add the missing file?

**Reply by vborchsh** [reply]
Work on it

**Reply by vborchsh** [reply]
btw there is an issue with git-flow.

I cannot synchronize dev branch in my fork, master only. Is there a way to do it?

**Reply by catkira** [reply]
maybe try
```
git switch dev
git fetch origin
git reset --hard origin/dev
git push -f
```

**Comment by zulfadlizainal** [comment]
this repo looks awesome. i have some experience in 5g. but doesnt have any experience in matlab. is there any scope i could contribute to the project?

**Reply by vborchsh** [reply]
Hi, thank you for participating!

Actually, the goal of the project is open-source implementation of MATLAB 5G NR toolbox functionality (fix me if I wrong @catkira). With possible future extensions and modifications.

So, you can start from [issues list](https://github.com/catkira/py3gpp/issues) OR check some `TODO` comments in the code OR move step by step through MATLAB functions just to translate them to Python.

I'd say that you definitely need Python's knowledge because the project is written in Python completely. Fortunately, both MATLAB and Python more or less friendly.

**Reply by catkira** [reply]
How about this issue
https://github.com/catkira/py3gpp/issues/50 ?

**Reply by zulfadlizainal** [reply]
thanks for the reply, let me try to read that and see whether its easy enough for beginner :)
